test/applyMiddleware.spec.ts

```diff
@@ -51,16 +51,11 @@ describe('applyMiddleware', () => {
     const spy = jest.fn()
     const store = applyMiddleware(test(spy), thunk)(createStore)(reducers.todos)
 
-    // the typing for redux-thunk is super complex, so we will use an as unknown hack
```

Comment:
This is why I installed `redux-thunk` as a dev dependency. I think it makes sense to ensure its compatibility here, seeing as the tests are using thunks.